app/Http/Controllers/Auth/AuthController.php

```diff
@@ -52,7 +51,7 @@ public function handleTwitterCallback()
             Auth::login($user);
         } else {
             Auth::login($user = User::create([
-                'name' => $twitter->name,
+                'name'       => $twitter->name,
```

Comment:
Thanks for the catch; vim and my .php_cs aren't playing nice. The config is there to collapse them, but it's not listening. I'll dig deep and fix :)